### ISSUE TITLE
Turn off grid lines when using plotly engine

### ISSUE THREAD

**Issue by alexanderbates** [natverse/nat#427]
Have an easy to use option to turn off gridlines when using the plotly engine @SridharJagannathan . Should maybe be the default. Useful esp. when plotting figures.

**Comment by alexanderbates** [comment]
This almost works, but somehow I loose transparency:

```plot3d(JRC2018F, col = "black", alpha = 0.1, plotengine="plotly")%>%
    plotly::layout(
           scene = list(xaxis = list(title = '', autorange = TRUE, showgrid = FALSE, zeroline = FALSE, showline = FALSE, autotick = TRUE, ticks = '', showticklabels = FALSE),
                        yaxis = list(title = '', autorange = TRUE, showgrid = FALSE, zeroline = FALSE, showline = FALSE, autotick = TRUE, ticks = '', showticklabels = FALSE),
                        zaxis = list(title = '', autorange = TRUE, showgrid = FALSE, zeroline = FALSE, showline = FALSE, autotick = TRUE, ticks = '', showticklabels = FALSE)
           )
    )```

**Comment by jefferis** [comment]
Closed by #432
